Consul/Consul.csproj

```diff
@@ -34,6 +34,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+    <PackageReference Include="ILMerge" Version="3.0.41" />
```

Comment:
The resulting NuGet now has a dependency on ILMerge. Is there a way to avoid that?

Reply:
Good catch, I forgot to check that. Yes, that's easy to fix, we just need to use `PrivateAssets="All"` attribute.